### PR TITLE
Updated README for navigable table of contents

### PR DESCRIPTION
**Pull request by Pallavi-Mukalla**
# Pull Request Format
## Type of PR
- **Add X in the box to specify the improvement type.**
- [ ] Bug fix
- [ ] Feature enhancement
- [x] Documentation update
- [ ] Other (specify): ___________  

## Description
This pull request updates README for navigable table of contents
## Screenshots / Videos (if applicable)
**Before:**
- it was difficult to navigate to various sections of the README

**After:**
- After the addition of README navigation became easier

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### Additional Context
[Provide any additional context about the changes, such as specific challenges you faced or decisions you made. This can help reviewers understand the rationale behind your changes.]

Thank you for reviewing my pull request!

### CODE REVIEW

**Comment by Pallavi-Mukalla** [comment]
please review my changes

**Comment by Pallavi-Mukalla** [comment]
please review my changes

File: README.md [hunk 1]
```diff
@@ -33,14 +43,15 @@ A <b> responsive and dynamic website </b> to showcase the best gaming accessorie
 
 
 
-## 💻 Technologies Used ( Frontend )
+## Technologies Used
```
Comment (swaraj-das) [inline README.md:46]: Your Readme formatting is good. But don't remove the emojis.

Comment (Pallavi-Mukalla) [reply README.md:46]: ok i will make the changes

**Comment by Pallavi-Mukalla** [comment]
i made the changes please review my changes

**Comment by Pallavi-Mukalla** [comment]
please review my changes

**Pull request closed without merging**